### ISSUE TITLE
"Remove Ban" button only works in english

### ISSUE THREAD

**Issue by Gastonite** [TheGeneticsGuy/Guild-Roster-Manager#39]
Because [the string "(Still in Guild)" is hardcoded](https://github.com/TheGeneticsGuy/Guild-Roster-Manager/blob/32ae452fb45b67c2e81ef09a06372a8ea7caa05b/GRM_Core.lua#L21493), the only way i found to use the "Remove Ban" button is changing the locale to English.

```
string.find ( fullName , GRM.L ( "(Still in Guild)" ) , 1 , true ) ~= nil then
```

**Comment by TheGeneticsGuy** [comment]
I just swapped a bunch of languages and didn't have any issues. What language are you using so I can try to recreate this.

**Comment by TheGeneticsGuy** [comment]
I can probably write the parsing better anyway with some regex, now that I look. This is some pretty old spaghetti code lol

**Comment by TheGeneticsGuy** [comment]
Alright I have this fixed in my 1.988 build. I'll release soon, just typing up changelog.

**Comment by Gastonite** [comment]
ok, I was using French language.

I am happy that my comments are used to improve this excellent addon =)
